corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -375,12 +377,14 @@ def test_bootstrap_if_needed(self, bootstrap_if_needed):
         bootstrap_if_needed.assert_called_once_with()
 
 
-class IndicatorPillowTest(TestCase):
+class IndicatorPillowTest(BaseRepeaterTest):
 
     @classmethod
     def setUpClass(cls):
         super(IndicatorPillowTest, cls).setUpClass()
         cls.config = get_sample_data_source()
+        cls.domain_obj.name = cls.config.domain
```

Comment:
I'm doing this, since `BaseRepeaterTest` set's up a domain for us, ready to use/handle repeaters